build.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
 
         // terra libraries
         crlPlatform = "0.2.0"
-        samClient = "0.1-9435410-SNAP"
+        samClient = "0.1-ffb0a89-SNAP" // "0.1-9435410-SNAP"
```

Comment:
Are there API changes that we might be missing in our (pretty stale) Sam fork that might require us to update our Verily Sam repo?

Reply:
I got a clean run of the CLI tests against `terra-verily-devel`, so I think we're fine for now. It's possible there are newer API endpoints that won't work, but the CLI tests aren't hitting any of them apparently.